CMakeLists.txt

```diff
@@ -532,18 +532,19 @@ set(QML_IMPORT_PATH "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}" CACHE PATH "QML director
 # whether unit tests should be build
 set (ENABLE_TESTS TRUE CACHE BOOL "Build unit tests?")
 if (ENABLE_TESTS)
-set( QT_USE_QTTEST  TRUE )
-enable_testing()
-# Adds some testing specific build targets e.g. make Experimental
-include(Dart)
-# Additional test configuration options e.g. max upload size of test report
-configure_file(
-  "${CMAKE_SOURCE_DIR}/cmake_templates/CTestCustom.cmake.in"
-  "${CMAKE_BINARY_DIR}/CTestCustom.cmake"
-  IMMEDIATE @ONLY)
-# For server side testing we have no X, we can use xvfb as a fake x
-# sudo apt-get install xvfb
-add_custom_target(check COMMAND xvfb-run --server-args=-screen\ 0\ 1024x768x24 ctest --output-on-failure)
+  set (PUSH_TO_CDASH FALSE CACHE BOOL "Determines whether test results should be pushed to CDASH site")
+  set(QT_USE_QTTEST TRUE)
+  enable_testing()
+  # Adds some testing specific build targets e.g. make Experimental
+  include(Dart)
+  # Additional test configuration options e.g. max upload size of test report
+  configure_file(
+    "${CMAKE_SOURCE_DIR}/cmake_templates/CTestCustom.cmake.in"
+    "${CMAKE_BINARY_DIR}/CTestCustom.cmake"
```

Comment:
@3nids hm, does this break submission of the nightlies?  Probably along with something else. For some reason ``CTestCustom.cmake`` is apparently not picked up anymore and submission is tried to a blank ``https://``.  Copying ``CTestCustom.cmake`` back to ``${CMAKE_SOURCE_DIR}`` helps at some places, but not everywhere (or on all branches).  Any hints?

Reply:
Submission to the dashboard? Have you defined CTEST_BUILD_NAME?

Reply:
Yes, to the dashboard.   The nightlies set `BUILDNAME` when configuring cmake.

Windows:

https://github.com/qgis/QGIS/blob/570efee7255d9201c460da095421eee2ea4882aa/ms-windows/osgeo4w/package-nightly.cmd#L120

Debian/Ubuntu:

https://github.com/qgis/QGIS/blob/570efee7255d9201c460da095421eee2ea4882aa/debian/rules#L84

And the osgeo4w testing builds work alike:

https://github.com/jef-n/OSGeo4W/blob/16ae5f88c12fe5678ca0c4c7efd2c352b0e20a91/src/qgis-dev/osgeo4w/package.sh#L131

But the results vary between platforms, branches and repos.

Reply:
It started: https://cdash.orfeo-toolbox.org/index.php?project=QGIS&date=2021-02-20